### ISSUE TITLE
Make doc.buyerReference usable

### ISSUE THREAD

**Issue by noseshimself** [dragotin/kraft#160]
1) There seems to be no UI element to access or modify the template element "doc.buyerReference".

2) There seems to be no "label.BUERREFERENCE" with a language-dependant content like "Ihr Zeichen"

**Comment by dragotin** [comment]
Yes, that field is mandatory in XRechnung, thus I added it. Indeed, it is useful.
